examples/web/index.html

```
@@ -1560,6 +1677,37 @@ <h1>Starkzap</h1>
+            <span
+              style="
+                font-size: 0.75rem;
+                font-weight: 600;
+                text-transform: uppercase;
+                letter-spacing: 0.05em;
+                color: var(--text-muted);
+              "
+              >Transaction History</span
+            >
+            <button
+              id="btn-bridge-tx-clear-completed"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)